gethoney/main.py

```diff
@@ -1,49 +1,51 @@
-from fastapi import FastAPI
-from gethoney.models import Honeypot
-from gethoney.crud import conn, curr
-import requests
-
-app = FastAPI()
-
-curr.execute(
-    """CREATE TABLE IF NOT EXISTS honeypots
-    (id INTEGER PRIMARY KEY AUTOINCREMENT, name TEXT, url TEXT, description TEXT)"""
-)
-
-# curr.execute(
-#     """CREATE TABLE IF NOT EXISTS logs
-#     (id INTEGER PRIMARY KEY, log_id INTEGER, name TEXT))"""
-# )
+from fastapi import Depends, FastAPI, HTTPException
 
+from gethoney.crud import Database
+from gethoney.models import Honeypot
 
-@app.get("/")
-def read_root():
-    return {"Hello": "World"}
+db_path = "../data/gethoney.db"

```

Comment:
We should change this to an environment variable, but that doesn't need to happen in this PR. we can do it in the next one, just keep that in mind